### DISCUSSION TITLE
Questtion about formatting

### DISCUSSION THREAD

**Discussion by mdziczkowski** [community/community#64835] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello. I had used in my project's Markdown README file the mentioned in the documentation

```
> [!IMPORTANT]
> sometext
```

syntax and on the preview it wasn't displayed in form like in the documentation (it has displayed the `[!IMPORTANT]` as code

**Comment by arafajs** [comment]
The syntax you used, [!IMPORTANT], is not a valid Markdown syntax. It is an extension of Markdown that is not supported by all Markdown parsers. The most common way to add emphasis to text in Markdown is to use asterisks or underscores. For example, the following Markdown will render as "This is important text":

`This is **important text**.`
You can also use the strong and em tags to add emphasis to text. For example, the following Markdown will render as "This is important text":

`This is <strong>important</strong> text.`

**Reply by mdziczkowski** [reply]
Strange. I had edited directly on GitHub (internal editor) and had used it as it was in the documentation.

Maybe it should be fixed to support it since it's documented? 🤔

By the way, is there (in GitHub) to create underscored text?

**Reply by arafajs** [reply]
@mdziczkowski 
You are right, the [!IMPORTANT] syntax is documented in the GitHub documentation, but it is not supported by the GitHub Markdown parser. I agree that it would be helpful if GitHub supported this syntax, as it would make it easier to create important text that stands out from the rest of the text.

**Reply by arafajs** [reply, marked as answer]
If you need to underline text in your GitHub Markdown file, you can use the HTML <u> tag. For example, the following Markdown will render as "This text is underlined":

`This text is <u>underlined</u>`.

**Reply by mdziczkowski** [reply]
This tag was given only as an example because there are many documented ones that aren't supported by the parser as well.

**Comment by GabBiswajit** [comment]
You got the answer or you have more questions?

**Reply by mdziczkowski** [reply]
Hello. I asked additional question above 😉

**Comment by GabBiswajit** [comment]
Hmm

**Reply by mdziczkowski** [reply]
I asked if there is a possibility to do underscored text in GitHub using markdown
